cmd/net-state/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/routes"
+	"github.com/storj/storage/boltdb"
+
+	"go.uber.org/zap"
+)
+
+func main() {
+	logger, err := zap.NewDevelopment()
+	if err != nil {
+		fmt.Println("zap logger failed to start")
+		return
+	}
+	defer logger.Sync()
+	logger.Info("serving on localhost:3000")
+
+	bdb, err := boltdb.New("test.db")
```

Comment:
not important for now, but you could make this a config value!

```
bdb, err := boltdb.New(viper.GetString("netstate.db.path"))
```

https://github.com/spf13/viper